File: basics/05-vowels-consonants/Solution.py
def count_vowels_consonants(s: str) -> tuple:
    vowels = set('aeiouAEIOU')
    v = c = 0
    for ch in s:
        if ch.isalpha():
            if ch in vowels:
                v += 1
            else:
                c += 1
    return v, c

File: basics/05-vowels-consonants/Solution.cpp
#include <string>
#include <utility>

std::pair<int, int> countVowelsConsonants(const std::string& s) {
    int v = 0, c = 0;
    std::string vowels = "aeiouAEIOU";
    for (char ch : s) {
        if (std::isalpha(ch)) {
            if (vowels.find(ch) != std::string::npos) v++;
            else c++;
        }
    }
    return {v, c};
}
